frontend/components/TeamsDropdown/_styles.scss

```diff
@@ -34,6 +34,7 @@
     top: 36px;
     border-radius: 6px;
     padding-right: 0;
+    margin: 0;
```

Comment:
Need to override the 1px here because there is no border to offset on this element.